### DISCUSSION TITLE
pydantic_settings package not supported

### DISCUSSION THREAD

**Discussion by bhutanict** [tiangolo/uvicorn-gunicorn-fastapi-docker#332] [Questions] [answered]
When I include the pydantic_settings package in my requirements.txt with the tiangolo/uvicorn-gunicorn-fastapi:python3.11 image, I am not able to build the project.

**Comment by tiangolo** [comment, marked as answer]
Now that Uvicorn supports managing workers with `--workers`, including restarting dead ones, there's no need for Gunicorn. That also means that it's much simpler to build a Docker image from scratch now, I updated the docs to explain it.

Because of that, I deprecated this Docker image: https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker#-warning-you-probably-dont-need-this-docker-image

So, you can build a Docker image from Python, from scratch (as explained in the docs), and that way you would be able to install everything more easily.
